store/index.js

```diff
@@ -31,14 +31,9 @@ export const mutations = {
     ;(state.cart = []), (state.cartUIStatus = "idle")
   },
   addToCart: (state, payload) => {
-    let itemfound = false
-    state.cart.forEach(el => {
-      if (el.id === payload.id) {
-        el.quantity += payload.quantity
-        itemfound = true
-      }
-    })
-    if (!itemfound) state.cart.push(payload)
+    let itemfound = state.cart.find(el => el.id === payload.id)
+    if (!itemfound) return void state.cart.push(payload)
+    itemfound.quantity += payload.quantity
```

Comment:
I feel like we could invert this, with incrementor first, and then you wouldn't need the weird void to throw away the return value. What do you think?

Reply:
Hi Sarah, I’m traveling and on mobile at the moment. Do you mean (pseudo):

if itemfound:
    increment quantity;
else:
    push onto array;

I agree that the void is a bit bleh. But you’re fine with .find()? That was my focus really.